build.gradle

```diff
@@ -211,7 +211,7 @@ project(':suro-s3') {
         compile project(':suro-localfile')
 
         compile 'net.java.dev.jets3t:jets3t:0.9.2'
-        compile 'com.amazonaws:aws-java-sdk:1.4.7'
+        compile 'com.amazonaws:aws-java-sdk-s3:1.9.3'
```

Comment:
I changed this because atlas plugin was pulling separate sdk.Platform is pulling uber jar but suro-localfile is not included in the platform, so I can control its dependencies on the suro-server side.